src/main/webapp/i18n/en/tutorialGroupImport.json

```diff
@@ -0,0 +1,75 @@
+                "description": "This dialog can be used to import tutorial groups and their registrations from a CSV file. Tutorial Groups will be created if they do not exist yet. Students will be deregistered from their current tutorial group and registered in the new one.",
+                "requirements": "Each row must always contain a value for the tutorial group title. If you want to not only create tutorial groups but also register students, the row must contain a value for either matriculation number or login. This is needed to identify the student. Values for last and first name are optional. The top row must specify the header!"
+            },
+            "supportedHeaderNames": {
+                "label": "Supported header names (versions with spaces, underscores, hyphens and uppercase letters are also supported): ",
+                "alwaysRequired": "Always required (to identify tutorial group): ",
+                "requiredIfStudentRegistration": "One of those is required if you want to register students (to identify students) :",
+                "alwaysOptional": "Always optional (just for display purposes): ",
+                "tutorialGroupTitle": "For tutorial group title: {{ headers }}",
+                "registrationNumber": "For matriculation number: {{ headers }}",
+                "login": "For login: {{ headers }}",
+                "firstName": "For first Name: {{ headers }}",
+                "lastName": "For last Name: {{ headers }}"
+            },
+            "errorMessages": {
+                "genericErrorMessage": "Import of tutorial groups failed!",
+                "withoutTitle": "The following rows miss a tutorial group title: ",
+                "noIdentificationInformation": "The following rows specify a user but do not contain a login or matriculation number: ",
+                "duplicatedLogins": "The following logins are assigned to multiple tutorial groups: ",
+                "duplicatedRegistrationNumbers": "The following matriculation numbers are assigned to multiple tutorial groups: "
+            },
+            "serverErrors": {
+                "notitle": "No tutorial group title",
+                "nologinorregistrationnumber": "No login or matriculation number",
+                "nouserfound": "No matching student found in course",
+                "multipleregistrations": "Multiple registrations in csv for student"
+            },
+            "csvFile": {
+                "label": "Select a .csv file",
+                "validationFailed": "Please repair the file, it is not considered valid due to the following error: ",
+                "parseCSVButton": "Parse CSV"
+            },
+            "fixedPlaceForm": {
+                "checkbox": {
+                    "label": "Do you want to specify a header and column to filter the rows?",
+                    "explanation": "Only those rows that have the specified value in the specified column, will be parsed from the CSV file."
```

Comment:
Optional: This explanation (and the ones below) could be a little bit more concrete.